ibis/common/grounds.py

```diff
@@ -167,10 +175,23 @@ def __reduce__(self):
         kwargs = dict(zip(self.argnames, self.args))
         return (self._reconstruct, (kwargs,))
 
+    # TODO(kszucs): consider to make a separate mixin class for this
+    def copy(self, **overrides):
+        kwargs = dict(zip(self.argnames, self.args))
```

Comment:
Ah, so then `dt.Whatever` instances will have a `copy` method and `__call__` becomes obsolete.